components/inputs/input-radio.js

```diff
@@ -84,6 +85,9 @@ class InputRadio extends InputInlineHelpMixin(SkeletonMixin(FocusMixin(PropertyR
 			.d2l-input-radio-supporting-visible {
 				display: block;
 			}
+			:host([_horizontal]) .d2l-input-radio-label {
```

Comment:
Instead of sprouting an attribute, can we just apply a CSS class to the label in `render()`?